cartridges/int_adyen_overlay/cartridge/scripts/util/AdyenHelper.ds

```diff
@@ -331,15 +338,13 @@ var __AdyenHelper : Object = {
  
         var jsonObject = __AdyenHelper.createAdyenRequestObject(recurringType, null, true);
 
-        var cardObject = __AdyenHelper.createCardObject({
-            FormType: 'account'
-        });
-        if ('card' in cardObject) {
-            jsonObject['card'] = cardObject.card;
-        }
-        if ('additionalData' in cardObject) {
-            jsonObject['additionalData'] = cardObject.additionalData;
-        }
+        var paymentMethodObject = __AdyenHelper.createCardObjectSF(false, 'account');
+        jsonObject['paymentMethod'] = paymentMethodObject;
+
+        jsonObject['recurring'] = {
+            'contract': 'ONECLICK'
+        };
```

Comment:
I believe this does not work with checkoutAPI. storeDetails = true should store it as ONECLICK. I think we need to add enableRecurring = true if recurring is enabled. How do we configure ONECLICK,RECURRING at the moment ?

Reply:
Changed it to enableOneClick=true